### PR TITLE
Update Apache Cassandra administration “cassandra_bur”

### PR DESCRIPTION
**Pull request by yellabelly**
Just missing a newline in the restore script between set and sort commands on line 49
`
set +esort -o ${keyspace_tablenames} ${keyspace_tablenames}
sort -o ${keyspace_tablenames} ${keyspace_tablenames}
`

### CODE REVIEW

**Comment by alexearnshaw** [comment]
@andreamussap Can you review this one for Martin please?

File: content/en/docs/cass_admin/cassandra_bur.md [hunk 1]
```diff
@@ -155,56 +142,55 @@ done
...
+    rm -rf /opt/cassandra/data/data/x9fa003e2_d975_4a4a_a27e_280ab7fd8a5_group_2/*
+    ```
 
 6. If you have other keyspaces to restore, return to step 4 and repeat. Otherwise, continue to the next steps.
 7. One at a time, start the Cassandra seed node, and then the other nodes, and wait for each to be in Up/Normal (`UN`) state in `nodetool status` before you proceed to the next node.
 8. Perform a full repair of the cluster as follows on each node one at a time:
 
     ```
-    nodetool repair -pr --full
+    odetool repair -pr --full
```
Comment (alexearnshaw) [inline content/en/docs/cass_admin/cassandra_bur.md]: @andreamussap I spotted this when having an initial look at the diff - the 'n' in 'nodetool repair' is missing.

**Comment by andreamussap** [comment]
Something went wrong with this PR, and the changes didn't show up in production. So, I reverted the changes by way of a new PR #189, and was able to apply the changes. Please see reverted PR here --> https://github.com/Axway/axway-open-docs/pull/189

**Comment by andreamussap** [comment]
### Thank you
We really appreciate your contribution and thank you for helping us to improve this documentation!
 
### Where to find the updated documentation
An Axway technical writer has reviewed and merged your PR, which is now available on the following sites:
Netlify (staging): https://axway-open-docs.netlify.com/docs/
Zoomin (production): https://docs.axway.com/bundle/axway-open-docs/
 
### Send us your feedback
If you have a moment, please complete the following 2 minute survey to give us your feedback on this process and help us make it even better:
https://www.surveymonkey.com/r/PJ5HQFD

**Pull request merged**